docs/data-operations-manual/Reference/Performance-db-tables-construction.md

```diff
@@ -0,0 +1,186 @@
+
+This is documentation that explains the construction of the tables in the [performance database](https://datasette.planning.data.gov.uk/performance/), including details on base tables, sources of columns.
+
+The Performance Database is designed to store and analyze performance-related metrics extracted from the source database [digital_land](https://datasette.planning.data.gov.uk/digital-land). The tables of this DB are:
+
+>1. `endpoint_dataset_issue_type_summary`
+>2. `endpoint_dataset_resource_summary`
+>3. `endpoint_dataset_summary`
+>4. `provision_summary`
+>5. `reporting_historic_endpoints`
+>6. `reporting_latest_endpoints`
+
+### 1. Table: [endpoint_dataset_issue_type_summary](https://datasette.planning.data.gov.uk/performance/endpoint_dataset_issue_type_summary)
+
+**Purpose**: To summarize issues associated with each dataset and endpoint.
+
+**Base Tables**:
+    - `issue`: Contains records of issues related to resources sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+    - `resource`: Holds information about the resources linked to the endpoints sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+    - `issue_type`: Defines types of issues sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
+
+**Columns**:
+- `organisation`: Extracted from the `provision` table.
+- `organisation_name`: Extracted from the `provision` table.
+- `cohort`: Extracted from the `provision` table.
+- `dataset`: Extracted from the `provision` table.
+- `collection`: Extracted from the `reporting_historic_endpoints` table.
+- `pipeline`: Extracted from the `reporting_historic_endpoints` table.
+- `endpoint`: Extracted from the `reporting_historic_endpoints` table.
+- `endpoint_url`: Extracted from the `reporting_historic_endpoints` table.
+- `resource`: Extracted from the `reporting_historic_endpoints` table.
+- `resource_start_date`: Extracted from the `reporting_historic_endpoints` table.
+- `resource_end_date`: Extracted from the `reporting_historic_endpoints` table.
+- `latest_log_entry_date`: Extracted from the `reporting_historic_endpoints` table.
+- `count_issues`: Count of issues from the `issue` table.
+- `date`: Current date when the query is executed.
+- `issue_type`: Type of issues from the `issue_type` table.
+- `severity`: Severity of issues from the `issue_type` table.
+- `responsibility`: Responsibility assigned to the issues from the `issue_type` table.
+- `fields`: Concatenated fields from `issue` table.
+
+
+### 2. Table: [endpoint_dataset_resource_summary](https://datasette.planning.data.gov.uk/performance/endpoint_dataset_resource_summary)
+
+**Purpose**: To summarize resources associated with endpoints, including mapping and non-mapping fields.
+
+**Base Tables**:
+    - `provision`: Contains information about organizations, cohorts, and datasets sourced from the [digital_land](https://datasette.planning.data.gov.uk/digital-land).
```

Comment:
provision table -detail about datasets provisioned for a particular Organisation.